tools/image_processing/yolo/yolo_training.xml

```diff
@@ -95,67 +106,103 @@
+                </output>
+                <output name="last_model">
+                    <assert_contents>
+                        <has_size value="5960100" delta="100"/>
+                    </assert_contents>
+                </output>
+                <output name="metrics_csv">
+                    <assert_contents>
+                        <has_n_lines n="11"/>
+                    </assert_contents>
+	            </output>
+	            <output name="metrics_plot" file="results_plot.png" compare="sim_size" />
         </test>
-
-	 <test>
-	    <param name="input_images" value="0001.jpg,0003.jpg,0006.jpg,0007.jpg,0013.jpg,0016.jpg,0018.jpg,0021.jpg"/>
-	    <param name="input_yolo" value="0001,0003,0006,0007,0013,0016,0018,0021" />
-	    <param name="model_url" value="https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11n-seg.pt" />
-	    <section name="training_params">
-                <param name="num_train" value="70" />
-                <param name="epochs" value="10" />
-                <param name="image_size" value="500" />
-                <param name="scale" value="0.8" />
-                <param name="degree" value="10.0" />
-                <param name="hsv_v" value="0.5"/>
-                <param name="init_lr" value="0.02" />
-                <param name="weight_decay" value="0.001" />
-	    </section>
-             <output name="best_model">
-                <assert_contents>
-                    <has_size value="5960100" delta="100"/>
-                </assert_contents>
-            </output>
-            <output name="last_model">
-                <assert_contents>
-                    <has_size value="5960100" delta="100"/>
-                </assert_contents>
-            </output>
-            <output name="metrics_csv">
-                <assert_contents>
-                    <has_n_lines n="11"/>
-                </assert_contents>
-	    </output>
-	    <output name="metrics_plot" file="results_plot1.png" compare="image_diff" />
+        <test>
+            <param name="input_images" value="0001.jpg,0003.jpg,0006.jpg,0007.jpg,0013.jpg,0016.jpg,0018.jpg,0021.jpg"/>
+            <param name="input_yolo" value="0001,0003,0006,0007,0013,0016,0018,0021" />
+            <conditional name="model_choice">
+                <param name="pretrained" value="https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11n-seg.pt"/>
+            </conditional>
+            <section name="training_params">
+				<param name="num_train" value="70" />
+				<param name="epochs" value="10" />
+				<param name="image_size" value="500" />
+				<param name="scale" value="0.8" />
+				<param name="degree" value="10.0" />
+				<param name="hsv_v" value="0.5"/>
+				<param name="init_lr" value="0.02" />
+				<param name="weight_decay" value="0.001" />
+            </section>
+                <output name="best_model">
+                    <assert_contents>
+                        <has_size value="5960100" delta="100"/>
```

Comment:
A bit of off topic, but would it actually make sense to include the option to run custom commands for output validation in a future release of Galaxy? Like, run some Python script to validate the produced output?